SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/TouchGrid.swift

```diff
@@ -84,53 +86,35 @@ class TouchGrid {
             removeSubTreeFromView(childnode)
         }
     }
+	
+
+	func beginScanning() {
+		scanningTimer = Timer.scheduledTimer(withTimeInterval: GlobalSettings.getUserSettings().scanSpeed.rawValue, repeats: true) { [weak self] _ in
```

Comment:
Can you break this into two lines by storing `GlobalSettings.getUserSettings().scanSpeed.rawValue` in a variable like `scanSpeed`? Just to make the line a little shorter.